data/twitter.csv

```diff
@@ -66,29 +74,22 @@ asn,handle
 23889,telecom_mu
 24904,knetfr
 25178,PCCWGlobal
+25255,dreioesterreich
 28403,RadiomovilDIPSA
 28573,NEToficial
 28704,proximus
 30722,VodafoneIT
+31423,PROMAX
+35228,O2
 36850,unc
 41998,NetComBW
 44944,Telenet
 46375,sonic
+47147,_ANEXIA
 47524,turksat
 47377,OrangeBeFR
 56478,HyperopticCS
 57933,IRISnet_team
 62044,Zscaler_Inc_EMEA
 134067,unitiwireless
-16591,GoogleFiber
-18209,ACT Fibernet
-8447,A1Telekom
-25255,dreioesterreich
-8412,magentatelekom
-1764,next_layer
-47147,_ANEXIA
-1853,ziduniwien
-21013,eww_Gruppe
-10139,SMARTCares
```

Comment:
Disappeared?

Reply:
No, I just sorted them according to ASN. The only twitter account I've removed is `10139,SMARTCares`, because there already was an account for that ASN. SMARTCares is customer support, LiveSMART is main account. Should I just leave both?

Reply:
In total: 12 twitter handles sorted, 1 removed as it was a duplicate, 2 added